developer-docs/dev_setup.md

```diff
@@ -1,215 +1,142 @@
 
-Please use the below template for your request:
+After having a token created your token will be shown under `Kubernetes` section of the `Connected Identities` tab on the sidebar.
 
-```
-Please can I be added to the UKHomeOffice organisation in quay.  
-Quay username: xxx
-Name: xxx xxx
-Team: My Project Team  
-```
-
-You have to login to be able to push docker images from your local machine to quay:
-
-```bash
-$ docker login quay.io
-```
-
-> **Please note** that quay doesn't create a password by default when you log in using Github. You need to create a password in order to log in into quay.
+#### Connecting to the cluster
 
-As all of our repositories are public you can then pull any of them. [Here are our Home Office quay repos](https://quay.io/organization/ukhomeofficedigital).
+In order to access a namespace you will need to configure `kubectl` to connect to our clusters - instructions and commands on how to set up kubectl can be found on the `Set up Kube config` button next to your kubernetes token under the `Connected Identities` page.
 
-## Artifactory access
-
-[Our private Artifactory is available here](https://artifactory.digital.homeoffice.gov.uk/artifactory/webapp/#/login).
-We use this for storing private docker images and other private artefacts (e.g. JARs, node modules, etc).
-
-When logging in please use the **HOD SSO** sign in option. To pull images from Artifactory you will need to do a docker login.
-Your username will be your digital email address and you can
-[generate an API key to use as your password here](https://artifactory.digital.homeoffice.gov.uk/artifactory/webapp/#/profile).
+Verify that your `kubectl` is configured properly by trying to list pods in the `developer-induction` namespace:
 
 ```bash
-$ docker login docker.digital.homeoffice.gov.uk
+$ kubectl --context=acp-notprod_ACP --namespace=developer-induction get pods
+No resources found.
 ```
+>The output of the command above may differ slightly depending on if there are pods deployed to the namespace.
 
-Images can be pulled with:
+## User agreement
 
-```bash
-$ docker pull docker.digital.homeoffice.gov.uk/aws-dsp:v0.4.7
-```
+Finally, please head over and read through our [SLA] documentation to familiarise yourself with the level of service which ACP provides to all members and projects - i.e. the support scope that we offer, support hours, and general escalation procedures.
 
-If you get this error instead:
-
-```
-Error: Status 400 trying to pull repository aws-dsp: "{\n  \"errors\" : [ {\n    \"status\" : 400,\n    \"message\" : \"Unsupported docker v1 repository request for 'docker'\"\n  } ]\n}"
-```
 
-You haven't logged in successfully.
+[https://access-acp.digital.homeoffice.gov.uk]:https://access-acp.digital.homeoffice.gov.uk
```

Comment:
[https://access-acp.digital.homeoffice.gov.uk] --> [Access ACP]
Something like this too?